### ISSUE TITLE
FadeInImage and png files

### ISSUE THREAD

**Issue by sm2017** [flutter/flutter#72412]
## Use case

I used `FadeInImage`, transparent part of png images is black and I think there is no way to change it

```dart
FadeInImage.memoryNetwork(
 placeholder: kTransparentImage,
 image: url,
)
```

## Proposal

A `backgroungColor` prop is needed to change background color

**Comment by pedromassangocode** [comment]
Hi @sm2017 
I fear that this may be an unexpected behavior.
Can you please provide your `flutter doctor -v` and a minimal complete reproducible code sample?
Thank you

**Comment by sm2017** [comment]
<Details>
<Summary>`flutter doctor -v`</Summary>

```
[√] Flutter (Channel stable, 1.22.5, on Microsoft Windows [Version 10.0.18363.1256], locale en-US)
    • Flutter version 1.22.5 at C:\flutter
    • Framework revision 7891006299 (6 days ago), 2020-12-10 11:54:40 -0800
    • Engine revision ae90085a84
    • Dart version 2.10.4


[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at C:\Users\PC\AppData\Local\Android\Sdk
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME = C:\Users\PC\AppData\Local\Android\Sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[!] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (1 available)
    • Android SDK built for x86 (mobile) • emulator-5554 • android-x86 • Android 10 (API 29) (emulator)

! Doctor found issues in 1 category.

```
</Details>


```dart
import 'package:flutter/material.dart';
import 'package:transparent_image/transparent_image.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    final title = 'Fade in images';

    return MaterialApp(
      title: title,
      home: Scaffold(
        appBar: AppBar(
          title: Text(title),
        ),
        body: Stack(
          children: <Widget>[
            Center(child: CircularProgressIndicator()),
            Center(
              child: FadeInImage.memoryNetwork(
                placeholder: kTransparentImage,
                image: 'https://www.pngitem.com/pimgs/b/505-5058955_sample-png.png',
              ),
            ),
          ],
        ),
      ),
    );
  }
}
```

**Comment by sm2017** [comment]
@pedromassangocode unfortunately I'm not able to reproduce it!! let me try again later

**Comment by sm2017** [comment]
@pedromassangocode I figured out that it is not related to flutter and it's my fault
